pkg/apis/broker/v1beta1/trigger_lifecycle.go

```diff
@@ -65,11 +65,11 @@ func (ts *TriggerStatus) PropagateBrokerStatus(bs *BrokerStatus) {
 
 	switch {
 	case bc.Status == corev1.ConditionUnknown:
-		ts.MarkBrokerUnknown(bc.Reason, bc.Message)
+		ts.MarkBrokerUnknown("Broker-"+bc.Reason, bc.Message)
```

Comment:
Separate discussion :)

I'm not sure why the `Broker` prefix is necessary. Isn't the condition type already `Broker`? So the reason and message will already be scoped to the broker's status. Can you give an example that illustrates why this is confusing?

Reply:
Makes sense; I hadn't considered the kubectl experience.